vaccine_feed_ingest/runners/il/state/normalize.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import os
+import pathlib
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.log import getLogger
+from vaccine_feed_ingest.utils.parse import location_id_from_name
+
+logger = getLogger(__file__)
+
+
+SOURCE_NAME = "il_state"
+
+
+def _get_id(site: dict) -> str:
+    return location_id_from_name(site["name"]) or "unknown"
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+
+    # there are multiple urls, vaccine, agency, health dept. etc
+    if website := site.get("website"):
+        return [schema.Contact(website=website)]
+
+    return None
+
+
+def try_lookup(mapping, value, default, name=None):
+    if value is None:
+        return default
+
+    try:
+        return mapping[value]
+    except KeyError as e:
+        name = " for " + name or ""
+        logger.warn("value not present in lookup table%s: %s", name, e)
+
+        return default
+
+
+def try_get_list(lis, index, default=None):
+    if lis is None:
+        return default
+
+    try:
+        value = lis[index]
+        if value == "none":
+            logger.warn("saw none value")
+        return value
+    except IndexError:
+        return default
+
+
+def _get_lat_long(site):
+    return schema.LatLng(
+        latitude=site.get("lat"),
+        longitude=site.get("long"),
+    )
+
+
+def _get_address(site):
+    return schema.Address(
+        street1=site.get("address"),
+        zip=site.get("zip"),
+        state="IL",
+        city=site.get("city"),
+    )
+
+
+# {"name": "ADAMS COUNTY HEALTH DEPARTMENT", "loc_type": "Public health provider - public health clinic", "address": "330 VERMONT ST", "city": "QUINCY", "zip": "62301", "county": "ADAMS", "website": "https://www.co.adams.il.us/government/departments/health-department/covid-19-vaccination-information", "lat": "39.9338798522949", "long": "-91.4105606079102", "pindetails": "<strong>Public health provider - public health clinic</strong><br/><a href='https://www.co.adams.il.us/government/departments/health-department/covid-19-vaccination-information'>ADAMS COUNTY HEALTH DEPARTMENT</a><br/>330 VERMONT ST<br/>QUINCY, IL  62301"}
```

Comment:
Remove this, or document why this is here

Reply:
my bad, this was a dev comment to more easiily compare what fields were available